Arrowgene.Ddon.GameServer/Handler/CraftStartEquipGradeUpHandler.cs

```diff
@@ -128,20 +127,10 @@ public override S2CCraftStartEquipGradeUpRes Handle(GameClient client, C2SCraftS
             {
                 equipItem.ItemId = equipItem.ItemId;
                 equipItem.EquipPoints = currentTotalEquipPoint;
-                UpdateCharacterItem(client, equipItemUID, equipItem, charid, updateCharacterItemNtc, CurrentEquipInfo);
+                Server.Database.UpdateItemEquipPoints(equipItemUID, currentTotalEquipPoint);
                 res = CreateEquipPointResponse(equipItemUID, addEquipPoint, currentTotalEquipPoint, goldRequired, IsGreatSuccess, CurrentEquipInfo, canContinue, dummydata);
             }
 
-            // TODO: Store saved pawn exp
-            S2CCraftCraftExpUpNtc expNtc = new S2CCraftCraftExpUpNtc()
-            {
-                PawnId = request.CraftMainPawnID,
-                AddExp = PawnExp,
```

Comment:
I assume the data behind PawnExp is still available in general? I do use it in my PR.